### ISSUE TITLE
Bug: retreat for evolved pokemon

### ISSUE THREAD

**Issue by z-mehta** [h0111in/PokemonGoBack#56]
need to check for  retreat with evolved pokemon, that is, when i play the retreat ability and swap the active card(basic pokemon) with the benched pokemon(stage 1 pokemon)..............the retreat property didnt happen properly: the basic pokemon disappear.........energy card goes to discard pile and the stage 1 pokemon stayed at the bench(it didn't move).

@h0111in look into the retreat ability when stage 1 pokemon is involved.

**Comment by h0111in** [comment]
thank you. it has been update.

**Comment by z-mehta** [comment]
yes retreat is working fine...............but if our active pokemon is a evolved pokemon(stage 1 pokemon),,,,,, n your click on retreat ..............then retreat will not happen .......it will give an error instead.
@h0111in look into it

**Comment by h0111in** [comment]
I have recently updated the code, please check it again and  give me the log.
